tensorflow_datasets/translate/opus.py

```diff
@@ -67,7 +67,7 @@ def __init__(self, name, description, homepage, url, languages):
     self.url = url
 
     language_pairs = []
-    for idx, source in enumerate(languages):
+    for idx, source in enumerate(sorted(languages)):
       for target in languages[idx + 1:]:
```

Comment:
you'd need to also sort here (or just do it earlier once)

Reply:
Done.